docs/GAME_FLOW.md

```diff
@@ -576,50 +666,34 @@ default gameflow for examples.
     </td>
   </tr>
   <tr valign="top">
-    <td>
-      <code>display_time</code>
-    </td>
+    <td><code>display_time</code></td>
     <td>Double</td>
   </tr>
   <tr valign="top">
-    <td>
-      <code>fade_in_time</code>
-    </td>
+    <td><code>fade_in_time</code></td>
     <td>Double</td>
   </tr>
   <tr valign="top">
-    <td>
-      <code>fade_out_time</code>
-    </td>
+    <td><code>fade_out_time</code></td>
     <td>Double</td>
   </tr>
   <tr valign="top">
-    <td>
-      <code>play_cutscene</code>
-    </td>
-    <td>
-      <code>cutscene_id</code>
-    </td>
+    <td><code>play_cutscene</code></td>
+    <td><code>cutscene_id</code></td>
     <td>Integer</td>
     <td>Plays the specified cinematic level (from the `cutscenes`).</td>
```

Comment:
Didn't catch this before, but `cutscenes` needs the `<code>` tags.

Reply:
Addressed